lib/statsd.js

```diff
@@ -535,7 +541,8 @@ function udsErrorHandler(client, lastSocketCreateTime) {
   if (newSocket) {
     client.socket.close();
     client.socket = newSocket;
-    maybeAddUDSErrorHandler(client, Date.now());
+    client.lastSocketCreateTime = Date.now();
```

Comment:
IMO it's strange to bolt random things onto this type. But maybe this isn't so random.

Reply:
I needed somewhere to cache the socket creation time because we didn't have access to it in `Client.prototype.sendMessage`. I'm open to move it elsewhere (currently leaning towards `client.socket` https://github.com/brightcove/hot-shots/pull/195/files#r537880161).